libpod/container_api.go

```diff
@@ -175,9 +173,27 @@ func (c *Container) StartAndAttach(ctx context.Context, streams *define.AttachSt
 		}
 	}
 
-	if err := c.prepareToStart(ctx, recursive); err != nil {
-		return nil, err
+	if c.state.State != define.ContainerStateRunning {
```

Comment:
could we attach to a container in the `ContainerStatePaused` state?

Reply:
I would think yes as conmon wouldn't care? But I just copied pre existing condition without thinking about it to much. If we attach to a paused container it could be confusing as no IO will be consumed, so a users could think the process/terminal hangs?

Reply:
we can deal with it in a separate PR.